packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -113,6 +113,24 @@
   </screenshots>
 
   <releases>
+   <release version="2023.05.0X-X" date="2023-05-03">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OpenStreetMap data as of March 29</li>
```

Comment:
(likely we'll update maps data again due to a possible bug)

• New OpenStreetMap data as of April 28
• Fixed OpenGL issues on some devices
• Search supports addr:interpolation addresses
• Faster rendering of subway layer
• Fixed wrong postcodes in Paris
• Fixed missing Wikipedia links
• Fixed car routing via roundabouts
• Improved English, German, Brazil translations
• Changed gate icon
• Changed Lyon subway icon
• Fixed US road shields
• Fixed priority of water areas
• Added camera shop, interior_decoration, antiques, art, cheese shop

Reply:
Done

Reply:
The date will be May 3.